pyvortex/src/lib.rs

```diff
@@ -2,21 +2,41 @@
 
 use array::PyArray;
 use expr::PyExpr;
+use pyo3::exceptions::PyRuntimeError;
 use pyo3::prelude::*;
 
 mod array;
 mod compress;
+mod dataset;
 mod dtype;
 mod encode;
 mod error;
 mod expr;
 mod io;
 mod python_repr;
+use lazy_static::lazy_static;
+use log::LevelFilter;
+use pyo3_log::{Caching, Logger};
+use tokio::runtime::Runtime;
+use vortex_error::{VortexError, VortexExpect as _};
+
+lazy_static! {
+    static ref TOKIO_RUNTIME: Runtime = Runtime::new()
```

Comment:
I *think* this is fine? though tagging @robert3005 @AdamGS who are more familiar with potential weirdness around python/async stuff